scripts/components/Interactions/HotspotNavButton.js

```diff
@@ -4,6 +4,7 @@ import React, {useCallback, useEffect, useRef} from 'react';
 import './NavigationButton.scss';
 import { H5PContext } from "../../context/H5PContext";
 import { scaleOpenContentElement } from '../../utils/open-content-utils';
+import {staticWidth, staticHeight, staticSceneWidth, staticSceneHeight} from "../Scene/SceneTypes/StaticScene";
```

Comment:
Formater dette likt som dei andre importene
```suggestion
import { staticWidth, staticHeight, staticSceneWidth, staticSceneHeight } from "../Scene/SceneTypes/StaticScene";
```